integration-tests/tests/billing-projects.js

```diff
@@ -40,9 +39,8 @@ const billingProjectsPage = (testPage, testUrl) => {
       await click(testPage, clickable({ text: 'Owners' }));
     },
 
-    selectProject: (billingProjectName, cloudPlatform = GCP) => {
-      const text = cloudPlatform === GCP ? `Google Cloud Platform${billingProjectName}` : `Microsoft Azure${billingProjectName}`;
-      return click(testPage, clickable({ text }));
+    selectProject: (billingProjectName) => {
```

Comment:
We lose integration coverage here of whether we showed the right icon, but that's actually already covered in a unittests-- `ProjectListItem.test.ts`.